Cod3rsGrowth.Web/wwwroot/controller/Lista.controller.js

```diff
@@ -0,0 +1,392 @@
+sap.ui.define([
+    "sap/ui/core/mvc/Controller",
+    "ui5/cod3rsgrowth/modelos/Repositorios/DataRepository",
+    "sap/ui/model/Filter",
+    "sap/ui/model/FilterOperator",
+    "ui5/cod3rsgrowth/modelos/Formatador",
+    "ui5/cod3rsgrowth/utilitarios/config",
+    "sap/ui/core/Fragment",
+    "sap/ui/core/routing/HashChanger",
+    "sap/ui/core/format/NumberFormat",
+    "sap/ca/ui/model/format/DateFormat",
+    "sap/ui/model/json/JSONModel"
+], (Controller,
+    DataRepository,
+    Filter,
+    FilterOperator,
+    Formatador,
+    config,
+    Fragment,
+    HashChanger,
+    NumberFormat,
+    DateFormat,
+    JSONModel) => {
+    "use strict";
+
+    return Controller.extend("ui5.cod3rsgrowth.controller.Lista", {
```

Comment:
todos os numeros ou strings devem estar em constantes se a string ou inteiro for utilizada em mais de um lugar vc coloca de forma global dentro da controller caso contrario so dentro da funcao,
e nao utiliza o var porque nao é recomendado, utiliza let